crates/ty_python_semantic/resources/mdtest/call/dunder.md

```diff
@@ -239,3 +239,37 @@ def _(flag: bool):
     # error: [possibly-unbound-implicit-call]
     reveal_type(c[0])  # revealed: str
 ```
+
+## Dunder methods cannot be looked up on instances
```

Comment:
Minor: This test here is very similar to the "Operating on instances" test above (search for `ThisFails`). Could we maybe add the "Class-level annotations with no value" part of this to the section above, instead?